sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/Utils.java

```diff
@@ -353,4 +362,44 @@ public static long macAddressToLong(String macAddress){
     public static String toHex(long number, int length) {
         return StringUtils.leftPad(Long.toHexString(number).toUpperCase(), length, "0");
     }
+
+
+    /**
+     * creates a copy of Scl element
+     *
+     * @param object object to copy
+     * @param clazz  class type of the object
+     * @param <T>    type of the object
+     * @return copy of the object
+     */
+    public static <T> T copySclElement(T object, Class<T> clazz) {
+        try {
+            JAXBContext jaxbContext = JAXBContext.newInstance("org.lfenergy.compas.scl2007b4.model");
+            JAXBElement<T> contentObject = new JAXBElement<>(new QName(clazz.getSimpleName()), clazz, object);
+            JAXBSource source = new JAXBSource(jaxbContext, contentObject);
+            Unmarshaller unmarshaller = jaxbContext.createUnmarshaller();
```

Comment:
Do we need to create a new `jaxbContext` and `unmarshaller` on every call of the method ? Could  `jaxbContext` and `unmarshaller` be static constants ?

Reply:
yes they could be static